pkg/genlib/fields/load.go

```diff
@@ -116,18 +174,25 @@ func getFromURL(ctx context.Context, srcURL string) ([]byte, error) {
 
 	client := &http.Client{}
 	resp, err := client.Do(req)
-
 	if err != nil {
+		defer func(r io.ReadCloser) {
```

Comment:
This is an odd construct?  defer a function and return immediately?

Reply:
yes sorry, the defer is not needed at all, doh! :)